src/mocks/handlers.js

```diff
@@ -1,36 +1,253 @@
 import { rest } from "msw";
 
 export const handlers = [
-    //  handles a POST add profile request
-    rest.post(
-        "../../api/api_endpoint/api/student/addResume/",
+    // handles a Post request to login
+    rest.post("../../api/api_endpoint/api/login/", (req, res, ctx) => {
+        return res(
+            ctx.json({
+                action: "Login",
+                message: "Verified",
+                user_type: ["student"],
+            })
+        );
+    }),
+
+    //  handles a POST requset to add student profile
+    rest.post("../../api/api_endpoint/api/student/profile/", (req, res, ctx) => {
+        return res(
+            ctx.json({
+                action: "Student Profile",
+                message: "Details Found",
+                details: {
+                    id: "190010036",
+                    resume_list: [{
+                        link: "https://storage.googleapis.com/cdc-backend-attachments/resume/190010036%2F8KIOT3PW1JIS718_CSE-V-SEM.pdf",
+                        name: "8KIOT3PW1JIS718_CSE-V-SEM.pdf",
+                    }, ],
+                    offers: [{
+                        designation: "Software Developer",
+                        company_name: "Make My Trip",
+                        application_id: "LLW4STE76GEJYOR",
+                    }, ],
+                    roll_no: 190010036,
+                    name: "Gowtham Sai",
+                    batch: "2019",
+                    branch: "CSE",
+                    phone_number: 9390291911,
+                    cpi: "9.15",
+                },
+            })
+        );
+    }),
+
+    // handles a GET request to get student Dashbord
+    rest.get(
+        "../../api/api_endpoint/api/student/getDashboard",
         (req, res, ctx) => {
             return res(
                 ctx.json({
-                    action: "Student Profile",
-                    message: "Details Found",
-                    details: {
-                        id: "190010036",
-                        resume_list: [{
-                            link: "https://storage.googleapis.com/cdc-backend-attachments/resume/190010036%2F8KIOT3PW1JIS718_CSE-V-SEM.pdf",
-                            name: "8KIOT3PW1JIS718_CSE-V-SEM.pdf",
+                    action: "Get Dashboard - Student",
+                    message: "Data Found",
+                    placements: [{
+                        id: "Q54IRZZMC3RP8F6",
+                        company_details_pdf_links: [{
```

Comment:
Add some real pdf links which will open

Reply:
For all pdf and images search some random pdf or image and keep those links

Reply:
PDF llink added